include/ac_t_umbrella.h

```diff
@@ -2,14 +2,39 @@
 #define AC_T_UMBRELLA_H
 
 #include "types.h"
-#include "m_actor.h"
+#include "ac_tools.h"
+#include "c_keyframe.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 extern ACTOR_PROFILE T_Umbrella_Profile;
 
+typedef void (*UMBRELLA_PROC)(ACTOR*);
+
+// Size == 0x1F4 (500)
+typedef struct t_umbrella_s {
+    TOOLS_ACTOR tools_class;
+    UMBRELLA_PROC action_proc;
+    int action;
+    f32 frame;
+    xyz_t scale_e;
+    xyz_t scale_kasa;
+    BOOL opened_fully;
```

Comment:
We never really put them, but it's personal preference if it's shown that is a bool